frontend/src/component/admin/network/NetworkTrafficUsage/BackendConnections.tsx

```diff
@@ -73,7 +73,7 @@ export const BackendConnections: FC = () => {
                         </TopRow>
                         <Bar
                             data={chartData}
-                            plugins={[customHighlightPlugin()]}
+                            plugins={[networkTrafficUsageHighlightPlugin]}
```

Comment:
The three charts in network traffic all share the same config here, so it seemed sensible to extract that configuration to a separate spot that they can all reference.